test/spec/ol/map.test.js

```diff
@@ -271,12 +271,47 @@ describe('ol.Map', function() {
         });
       });
     });
+
+    describe('#getEventPixel', function() {
+      it('works with touchend events', function() {
+        var target = document.createElement('div');
+        target.style.position = 'absolute';
+        target.style.top = '10px';
+        target.style.left = '20px';
+        target.style.width = '800px';
+        target.style.height = '400px';
+
+        document.body.appendChild(target);
+
+        var map = new ol.Map({
+          target: target
+        });
+
+        var browserEvent = new goog.events.BrowserEvent({
+          type: 'touchend',
+          target: target,
+          changedTouches: [{
+            clientX: 100,
+            clientY: 200
+          }]
+        });
+        var position = map.getEventPixel(browserEvent.getBrowserEvent());
+        // 80 = clientX - target.style.left
+        expect(position[0]).to.eql(80);
+        // 190 = clientY - target.style.top
+        expect(position[1]).to.eql(190);
+
+        document.body.removeChild(target);
```

Comment:
Shouldn't you do that in an `afterEach` so it is always done, even if an exception occurs in the middle of the test? You'll also want to do the `appendChild` in `beforeEach` I guess.